recipes-robot/fs-overlay/files/etc.mount

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
Correct me if I'm wrong, but this creates an overlay for the entirety of `/etc`, right? But I don't think we _want_ to overlay all of etc, because then a user or code can edit it, in a way that may not be safe to update. We really only want to carve out _extremely specific_ directories and possibly even specific files to be writable, and then only if we really cannot figure out any other way to make them able to be read-only.